plugins/inputs/kafka_consumer/kafka_consumer.go

```diff
@@ -133,14 +137,17 @@ func (k *Kafka) receiver() {
 				k.acc.AddError(fmt.Errorf("Kafka Consumer Error: %s\n", err))
```

Comment:
Was thinking of adding a tag here, probably 'E!'. Make sense? And should it end in a newline?